### PR TITLE
figure 2

### PR DESCRIPTION
**Pull request by sumedha-k**
@aarmey here's figure 2
image

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Please just replace the Fig. 2 file.

**Comment by sumedha-k** [comment]
@aarmey do you know what this error is? I'm not getting it when I ran it locally.

**Comment by sumedha-k** [comment]
This is what the new figure looks like:
image

**Comment by sumedha-k** [comment]
However I'm not sure that this is how we want to commute the error bars. 
`p33 = np.transpose(np.dot(p33_scores, p33_loadings))
 p66 = np.transpose(np.dot(p66_scores, p66_loadings))`

File: deconv/figures/figure2.py [hunk 1]
```diff
@@ -1,52 +1,36 @@
...
+    p66 = np.transpose(np.dot(p66_scores, p66_loadings))
 
-    # Add subplot labels
-    subplotLabel(ax)
 
-    return f
+    for i in range(len(measurement_glycan_matrix)):
+        ax[i].errorbar(glycans, measurement_glycan_matrix[i], yerr=[p33[i], p66[i]], fmt='o')
+        ax[i].set_title(l[i])
+        ax[i].set_xticklabels(glycans, rotation=90)
```
Comment (aarmey) [inline deconv/figures/figure2.py:36]: Your figure function has to return `f`. Once that's added I expect this will build.

**Review by aarmey — CHANGES_REQUESTED** [review]
One edit, then this should be ready to merge.

**Comment by sumedha-k** [comment]
@aarmey Returning f in the previous commit seemed to fix it. But I just pushed another commit to delete figure 11 and the build failed on figure 2 again?

**Comment by aarmey** [comment]
No, the build failed on Fig. 11. Fig. 2 just was already running and finished.

**Comment by aarmey** [comment]
I fixed the issue of the quantile amount versus the error bar. Also, the quantiles have to be solved *after* reconstructing all the samples. Will merge this if it builds.

**Pull request merged**